torch/multiprocessing/spawn.py

```diff
@@ -134,21 +143,23 @@ def join(self, timeout=None):
             # Return whether or not all processes have been joined.
             return len(self.sentinels) == 0
 
-        # Assume failure. Terminate processes that are still alive.
```

Comment:
We should keep the mention here that we assume a failure happened. And we're just cleaning things now

Reply:
I think it's not really an "assumption"? an error did happen.  Added this note above that we're cleaning up after error.